### ISSUE TITLE
Dashboard Menu grouping

### ISSUE THREAD

**Issue by EZarnosky** [Hellowlol/HTPC-Manager#232]
it would be nice to build sub menus off the HTPC Manager menu bar to reduce clutter. Example menu item "Downloaders" could include "Transmission" & "NZBGet". Or "Media Systems" could include "Samsung TV", "Plex", "Kodi".

**Comment by Hellowlol** [comment]
I agree, my menu names are very short to prevent the menu from wrapping. I'm not sure how to solve this..

**Comment by thatbritguy** [comment]
I've been thinking about this a lot whilst creating the themes and tinkering with the other UI elements.  Think I've come up with a solution but need a bit of time to play around and experiment with it to see if it works in practice.

**Comment by EZarnosky** [comment]
Thats awesome. If you need testers, just let me know :)

**Comment by EZarnosky** [comment]
Hi, any progress on this? Other Links does have grouped links in  the menu, it would be nice for the other apps also.

**Comment by Hellowlol** [comment]
None in afraid. Im but sure how to handle this and it dont really interest me :P
